aggregations/db_tables/daily_new_accounts_by_app_count.py

```diff
@@ -0,0 +1,89 @@
+import datetime
+
+from . import DAY_LEN_SECONDS, daily_start_of_range, time_json
+from ..periodic_aggregations import PeriodicAggregations
+
+
+class DailyNewAccountsByAppCount(PeriodicAggregations):
+    def dependencies(self) -> list:
+        return ['near_ecosystem_entities']
+
+    @property
+    def sql_create_table(self):
+        return '''
+            CREATE TABLE IF NOT EXISTS daily_new_accounts_by_app_count
+            (
+                collected_for_day  DATE NOT NULL,
+                app_id TEXT NOT NULL,
+                new_accounts INTEGER NOT NULL,
+                PRIMARY KEY (collected_for_day, app_id )
+            );
+            CREATE INDEX IF NOT EXISTS daily_new_accounts_by_app_count_idx
+                ON daily_new_accounts_by_app_count (collected_for_day, new_accounts DESC)
+        '''
+
+    @property
+    def sql_drop_table(self):
+        return '''
+            DROP TABLE IF EXISTS daily_new_accounts_by_app_count
+        '''
+
+    @property
+    def sql_select(self):
+        all_apps = '''
+            SELECT token, slug as app_id
```

Comment:
1. What is the meaning for `token`?
2. Given you renamed `slug` to `app_id`, I feel that the original table might have the column named as `app_id` then rather than `slug`. What do you think?

Reply:
Token is a keyword driving functionality of sql being used on the join.  I renamed to contract.  But it is being referenced by position further down in the code, so the name doesn't matter, the position does.  

Slug is the name used by the ecosystem team.  It is where you can find the "entity"  The page URL that follows after near.org/ecosystem/  .  This will be used for apps, nft & guilds I believe, so let's  leave that generalized in the entity table.